deb-config/spinnaker/bin/secrets

```diff
@@ -1,3 +1,29 @@
 #!/bin/bash
 
-echo "We need to pull secrets using this"
+# This file depends on how your company manages secrets.
+
+
+echo "/opt/spinnaker/bin/start will run this during startup to fetch secrets."
+exit 0;
+
+
+
+# This is an example of using a private s3 bucket.
+aws s3 cp s3://secrets-bucket/spinnaker/production/dockerhub.password /opt/spinnaker/secrets/
+aws s3 cp s3://secrets-bucket/spinnaker/production/kubernetesConfig /opt/spinnaker/secrets/
+
+
+# Example of jenkins username and password. -local.yml will read variables from /opt/spinnaker/env/resolved.env
+aws s3 cp s3://secrets-bucket/spinnaker/production/jenkins.credentials /opt/spinnaker/secrets/
+cat /opt/spinnaker/secrets/jenkins.credentials >> /opt/spinnaker/resolved.env
```

Comment:
The dest path for the aggregated resolved.env is wrong. But otherwise looks good to me, ready to merge